configs/components/echam/echam.datasets.yaml

```diff
@@ -51,58 +54,100 @@ forcing_sources:
+                        to: 2014
+                "${forcing_dir}/volcano_aerosols/strat_aerosol_sw_${resolution}_2014.nc":
+                        from: 2015
+
         "1950volcir":
                 "${forcing_dir}/volcano_aerosols/strat_aerosol_ir_${resolution}_1950.nc"
+
         "1950volcsw":
                 "${forcing_dir}/volcano_aerosols/strat_aerosol_sw_${resolution}_1950.nc"
+
         # greenhouse
+        # TODO: currently only r0008 contains the greenhouse_ssp* files
         "histgreenh": "${greenhouse_dir}/greenhouse_historical.nc"
         "greenhouse_hist_scenario":
                 "${greenhouse_dir}/greenhouse_historical.nc":
                         to: 2014
                 "${greenhouse_dir}/greenhouse_${scenario}.nc":
                         from: 2015
+        greenhouse_cmip5_scenario:
+                "${greenhouse_dir}/greenhouse_historical.nc":
+                        to: 2014
+                "${greenhouse_dir}/greenhouse_{scenario_lowercase}.nc":
+                        from: 2015
 
+        # TODO: greenhouse files differ
+        # greenhouse
         "1percCO2greenh": "${greenhouse_dir}/greenhouse_1pctCO2.nc"
+        #"histgreenh": "${greenhouse_dir}/greenhouse_historical.nc"
+        # - use any SSP file for the historical scenario to have GHG values for year 2015,
+        #   necessary for Echams year+1 interpolation for last historical year 2014
+        # - all historical years <= 2014 are identical throughout all SSP GHG files
+        # - year 2015 CO2 values: 126:399.9492, 245:399.9491, 370:399.9484, 585:399.9488 ppm
+        #   --> doesn't really matter which is used. T. Semmler and C. Danek used 370 for their CMIP6 simulations
+        # TODO: these files (ssp greenhouse) are missing in release -> REMOVED
+        
         # radiation
         "histswflux":
                 "${input_dir}/solar_irradiance/swflux_14band_1850.nc":
                         to: 1850
                 "${input_dir}/solar_irradiance/swflux_14band_@YEAR@.nc":
                         from: 1851
+        
         "1950swflux": "${input_dir}/solar_irradiance/swflux_14band_1950.nc"
+        # TODO: years are different
         # ozone
-        "histozone":
-                "${forcing_dir}/ozone/${resolution}_ozone_historical_1850.nc":
-                        to: 1850
-                "${forcing_dir}/ozone/${resolution}_ozone_historical_@YEAR@.nc":
-                        from: 1851
-                        to: 2008
         "ozone_hist_scenario":
                 "${forcing_dir}/ozone/${resolution}_ozone_historical_1850.nc":
                         to: 1850
                 "${forcing_dir}/ozone/${resolution}_ozone_historical_@YEAR@.nc":
                         from: 1851
                         to: 2014
+                # scenario: ssp126, ssp119, ssp245, ssp370, ssp585
                 "${forcing_dir}/ozone/${resolution}_ozone_${scenario}_@YEAR@.nc":
                         from: 2015
```

Comment:
make the years depeding on r0007/r0008?

Reply:
It looks to me that r8 extends r7. r8 contains both the historical and the future data. I calculated the checksums of the same file names in both directories and they seem to be the same. So, let's go with r8 since it has more data. 
I have no idea they are storing the duplicate data in a different directory 🤔